scipy/stats/stats.py

```diff
@@ -3486,6 +3486,8 @@ def kendalltau(x, y, initial_lexsort=None, nan_policy='propagate'):
     >>> x1 = [12, 2, 1, 12, 2]
     >>> x2 = [1, 4, 7, 1, 0]
     >>> tau, p_value = stats.kendalltau(x1, x2)
+    >>> # Cross-check with R:
+    >>> # cor.test(c(12,2,1,12,2),c(1,4,7,1,0),method="kendall",exact=FALSE)
```

Comment:
Yeah, this is better as an actual test. 

Meanwhile, am I doing something wrong --- 

```
> cor.test(c(12,2,1,12,2),c(1,4,7,1,0),method="kendall",exact=FALSE)

	Kendall's rank correlation tau

data:  c(12, 2, 1, 12, 2) and c(1, 4, 7, 1, 0)
z = -1.0742, p-value = 0.2827
alternative hypothesis: true tau is not equal to 0
sample estimates:
       tau 
-0.4714045 
```

and 
```
In [1]: from scipy.stats import kendalltau

In [2]: kendalltau([12, 2, 1, 12, 2], [1, 4, 7, 1, 0])
Out[2]: KendalltauResult(correlation=-0.47140452079103162, pvalue=0.28274545993277478)
```

So ISTM, `stats.kendalltau`'s p-value is different?